subsys/usb/class/bt_h4.c

```diff
@@ -41,13 +41,13 @@ static K_FIFO_DEFINE(tx_queue);
 
 /* Data size needed for CMD and ACL buffers */
 #define BUF_SIZE MAX(BT_BUF_RX_SIZE, BT_L2CAP_BUF_SIZE(BT_L2CAP_MTU))
-NET_BUF_POOL_DEFINE(tx_pool, BUF_COUNT, BUF_SIZE, sizeof(u8_t), NULL);
+NET_BUF_POOL_DEFINE(rx_pool, BUF_COUNT, BUF_SIZE, sizeof(u8_t), NULL);
 
-#define BT_H4_OUT_EP_ADDR		0x01
-#define BT_H4_IN_EP_ADDR		0x81
+#define BT_H4_OUT_EP_ADDR               0x01
```

Comment:
Changes here (commit e81d3694161617c1ef29b11ed293ca9627033e03) looks like a rebase accident

Reply:
Right, this was meant to be a fixup to the previous patch, will fix it.

Reply:
Fixed.